pybaseball/datasources/fangraphs.py

```diff
@@ -153,7 +151,7 @@ def fetch(self, start_season: int, end_season: Optional[int] = None, league: str
         }
 
         return self._validate(
-            self._postprocess( 
+            self._postprocess(
```

Comment:
_sigh_ we really do need an autoformatter
(you're 100% in the right to fix this, but avoiding the stupid minimal diffs would be great

Reply:
To be fair, this wasn't the autoformatter as much as it was bugging me and I have a low grade case of OCD. 😆 

Does that make me the autoformatter?